sources/mongodb/__init__.py

```diff
@@ -0,0 +1,85 @@
+"""Source that loads collections form any a mongo database, supports incremental loads."""
+
+from typing import List, Optional, Iterable, Any
+
+import dlt
+from dlt.extract.source import DltResource
+
+from .helpers import (
+    collection_documents,
+    client_from_credentials,
+    MongoDbCollectionConfiguration,
+    MongoDbCollectionResourceConfiguration,
+)
+
+@dlt.source
+def mongodb(
+    connection_url: str = dlt.secrets.value,
+    database: str = dlt.config.value,
+    collection_names: Optional[List[str]] = dlt.config.value,
+    incremental: Optional[dlt.sources.incremental[Any]] = dlt.config.value,
```

Comment:
you do not need this at source level. people will configure that per table

Reply:
It will never run on source level? I based it on the sql one, should I remove it? Sorry I am not sure if I understood this part.